packages/components/src/components/DetailsPanel.vue

```diff
@@ -185,6 +193,21 @@ export default {
     openStatsPanel() {
       this.$emit('openStatsPanel');
     },
+    analysisFindingSelection(findingObject) {
+      const finding = findingObject.resolvedFinding && findingObject.resolvedFinding.finding;
+      if (!finding) return;
+
+      if (finding.checkId === 'n-plus-one-query') {
+        this.$store.commit(SET_HIGHLIGHTED_EVENTS, finding.relatedEvents);
+        this.$store.commit(SET_VIEW, VIEW_FLAMEGRAPH);
+
+        const eventToFocus = finding.relatedEvents.find((relatedEvent) => !relatedEvent.sql_query);
```

Comment:
Every finding can have relatedEvents. So does this need to be hard coded to N + 1 query?

Reply:
@dustinbyrne @brikelly and I met on Friday and thought that N+1 query findings would be a good place to start improving how we display findings in AppMaps. We figured that the optimal way to display a finding in the map would be different for different types of findings. There will probably be overlap, though. For example, we talked about how other types of performance findings might be displayed in a similar way to N+1 query findings. But for now this seems like a good place to start making changes.